README.md

```diff
@@ -409,29 +405,53 @@ details about formats.
 
 ### 3. Debugging HarbourBridge
 
-The HarbourBridge tool can fail for a number of reasons
-
--   Unparsable pg_dump: HarbourBridge uses the
-    [pg_query_go](https://github.com/lfittl/pg_query_go) library. It is possible
-    that the pg_dump output is corrupted or uses features that aren't parseable.
-    Parsing errors should generate an error message of the form `Error parsing
-    last 54321 line(s) of input`.
-
--   Credentials problems: HarbourBridge uses standard Google Cloud credential
-    mechanisms for accessing Cloud Spanner. If this is mis-configured, you may
-    see errors containing "unauthenticated", or "cannot fetch token", or "could
-    not find default credentials". You might need to run `gcloud auth
-    application-default login`. See the [Before you begin](#before-you-being)
-    section for details.
-
--   Can't create database: the error message printed by the tool should help
-    identify the cause. It could be an API permissions issue. For example, the
-    Cloud Spanner API may not be appropriately configured. See
-    [Before you begin](#before-you-being) section for details. Alternatively,
-    you have have hit the limit on the number of databases per instances
-    (currently 100). This can occur if you re-run the HarbourBridge tool many
-    times, since each run creates a new database. In this case you'll need to
-    [delete some databases](https://cloud.google.com/spanner/docs/getting-started/go/#delete_the_database).
+The HarbourBridge tool can fail for a number of reasons.
+
+#### 3.1 No space left on device
+
+HarbourBridge needs to read the pg_dump output twice, once to build a schema and
+once for data ingestion. When pg_dump output is directly piped to HarbourBridge,
+`stdin` is not seekable, and so we write the output to a temporary file. That
+temporary file is created via go's ioutil.TempFile. On many systems, this
+creates a file in `/tmp`, which is sometimes configured with minimal space. A
+simple workaround is to separately run pg_dump and write its output to a file in
+a directory with sufficient space.  For example, if the current working
+directory has space, then:
+
+```sh
+pg_dump > tmpfile
+harbourbridge < tmpfile
+```
+
+Make sure you cleanup the tmpfile after HarbourBridge has been run. Another
+option is to set the location of go's TempFile e.g. by setting the `TMPDIR`
```

Comment:
*Go's

Reply:
Done.
And fixed a couple of other as well.